apps/e2e/modals-storybook/src/app/modal/modals/modal-basic.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+import { SkyModalInstance } from '@skyux/modals';
+
+@Component({
+  selector: 'app-modal',
+  templateUrl: './modal-basic.component.html',
+})
+export class ModalBasicComponent {
+  #instance: SkyModalInstance;
+  constructor(instance: SkyModalInstance) {
```

Comment:
Our convention for classes is to put public properties at the top, then private properties, then the constructor, then public methods, and finally private methods. Also you should have blank lines between each of these sections and in between methods.